deploy/api.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+export ANALYTICS_REPORTS_PATH=reports/api.json
+
+# Gov Wide
+$HOME/bin/analytics --verbose --write-to-database --output /tmp
```

Comment:
@jmhooper OK, that points to a `--silent` option. What do you think of that as a takeaway for a separate PR?